src/net/sourceforge/kolmafia/request/BasementRequest.java

```diff
@@ -1192,8 +1184,8 @@ public static long getBasementTestCurrent() {
     return (long) BasementRequest.basementTestCurrent;
   }
 
-  public static long getBasementTestValue() {
-    return (long) BasementRequest.basementTestValue;
```

Comment:
This is the cast that the security scanning didn't like.  This function is only called in BasementDecorator and the result is only used in a String so I am not expecting the change to create an issue.

Reply:
Is the security scanning happy with instead using Double.longValue()? Or Math.round()?

I mention this because printing a double can be much more verbose.

Reply:
I don't know how to run the scanning locally.  If it is easy I will try.  Otherwise I'm fine with trying Double.longValue().

Reply:
And the change was made.